src/cryptoadvance/specter/templates/wallet/send/new/wallet_send.jinja

```diff
@@ -190,183 +220,103 @@
 			}
 		})
 
+
 		function addRecipient(addr, amount, amount_unit, label) {
```

Comment:
This functionality would be relevant for extensions which integrate with exchanges. So if you want to sell some bitcoin (stupid idea anyway) you could fill a form automatically with the address of the exchange.
However it's currently not used and that already since month and so it might make sense to make it anew when it's necessary again.

So feel free to purge this as needed.

Reply:
@relativisticelectron I think this is the only remaining task. 

This 
```
		{% if fillform %}
			{% for addr, amount, amount_unit, label in recipients %}
			addRecipient("{{ addr }}", {{ amount }}, "{{ amount_unit }}", "{{ label }}");
			{% endfor %}
		{% else %}
			addRecipient("", 0, "btc", "");
```
probably will reduce to just` addRecipient()`, no? Please also adjust the other occurrences ` addRecipient()` then (and it particular the definition of `add Recipient()`in `wallet_send`).

Reply:
The functionality of `fillform` should be independent of this PR.  Would it not be better to do the removal/replacement  in a different PR?  `fillform` also occurs in  https://github.com/cryptoadvance/specter-desktop/blob/7cf5b7637dedb40ee5f77b45270d5e1d351c2012/src/cryptoadvance/specter/server_endpoints/wallets.py#L597

Reply:
There is a comment, there, too: `# TODO: Not yet used. Remove if the use case doesn't happen.`

We can handle the `fillform` stuff in a different PR, but IMO this PR is defining `addRecipient()` in a new way and should have the correct arguments. In Python the code would fail, but JS is using "null" if undefined arguments are passed.

Reply:
OK, we should check how / whether fillfrom=True is really used [there](https://github.com/cryptoadvance/specter-desktop/blob/7cf5b7637dedb40ee5f77b45270d5e1d351c2012/src/cryptoadvance/specter/server_endpoints/wallets.py#L558) and whether it breaks sth. with rbf

Reply:
Yes it is, I just checked. And I am currently implementing that the arguments in `addRecipient` are used.

Reply:
85a4777c81ed213435ad3240014c7395d3b19d88 enables the use of the arguments in `addRecipient`